pkgs/by-name/ve/vegastrike-utcs-data/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, vegastrike
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vegastrike-utcs-data";
+  inherit (vegastrike) version;
+
+  src = fetchFromGitHub {
+    owner = "vegastrike";
+    repo = "Assets-Production";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-XGDg3s8VAtJsO2HnbjA3Yt42euBFH79hIFfSh81jafk=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    cp -r $src $out
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Vega Strike - Upon The Coldest Sea - Assets";
+    homepage = "https://www.vega-strike.org";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ TheBrainScrambler ];
+    platforms = platforms.linux;
```

Comment:
On the GH repo of VegaStrike, I see also support for Windows and macOS. Would it be also possible to add packages for those? I know that it would require compilation by hand.